ApplicationConfig/Providers/SiteAccesses.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\ApplicationConfig\Providers;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use EzSystems\PlatformUIBundle\ApplicationConfig\Provider;
+
+class SiteAccesses implements Provider
```

Comment:
I know but we can take the first one as the default language when creating a content (basically same logic as in https://github.com/ezsystems/PlatformUIBundle/pull/581/commits/7a8c177f5a8ffa8bfe566adfb4910dd3fba3d14b#diff-4e897328f624be930d091d1ce688a3ffR233 but here instead)